wcivf/apps/people/templates/people/includes/_person_about_card.html

```diff
@@ -1,17 +1,18 @@
+{% load i18n %}
 {% if object.wikipedia_bio or object.favourite_biscuit %}
     <div class="ds-card">
         <div class="ds-card-body">
             <h2 class="ds-candidate-name ds-h3">
-                About {{ object.name }}
+                {% blocktrans with name=object.name %}About {{ object.name }}{% endblocktrans %}
```

Comment:
I think a `blocktrans` tag is necessary to translate "About", but a name is not translatable is the `name=object.name` necessary?

Reply:
1. We can't use `object.name`, because we can only translate simple variables. So this should be `name`.
2. When making the vars, it's useful to think about what the translators will see in their interface. Here all they will see is that they need to translate `"about [name]"`, but this isn't as useful as e.g `"about [person_name]"`. Or `"about [ward_name]"`. e.g, can you try to describe what the vars are in the name, just in case that's useful?

If 2. isn't easy in the var names, we can use [context markers](https://docs.djangoproject.com/en/3.2/topics/i18n/translation/#contextual-markers) or [comments](https://docs.djangoproject.com/en/3.2/topics/i18n/translation/#comments-for-translators-in-templates)